packages/nimble-components/src/table/tests/table-keyboard-navigation.spec.ts

```diff
@@ -1422,6 +1426,28 @@ describe('Table keyboard navigation', () => {
                     expect(blurSpy).toHaveBeenCalledTimes(1);
                     expect(currentFocusedElement()).not.toBe(cellContent);
                 });
+
+                it('and then the cell updates to no longer have tabbableChildren, the cell is focused instead', async () => {
```

Comment:
I presume if we went with this approach (which I think I'm fine with), that there are some other tests we still need to add, yes? Namely, that after the tabbableChildren change (but not completely removed, as this test covers that) that tabbing (and shift-tabbing) behave as expected.

Reply:
Agreed, those would be good tests to add too.